src/adapter/src/coord.rs

```diff
@@ -584,10 +585,9 @@ impl Coordinator {
                     .storage_ids
                     .extend(replica.config.compute.logging.source_ids());
 
-                self.controller
-                    .create_replica(instance.id, replica_id, replica.config)
-                    .await?;
+                replicas_to_start.push((instance.id, replica_id, replica.config));
             }
+            self.controller.create_replicas(replicas_to_start).await?;
```

Comment:
This is still one invocation per cluster. Is there a reason not to have a single invocation that creates the replicas for all clusters at once?

Reply:
I misread the code and thought that this did this; Ill switch it to that!